From 778d9230cc277b0b25bdfeec9177a197b46891b8 Mon Sep 17 00:00:00 2001
From: sadhansx <sadhanx.siddha.mohanty@intel.com>
Date: Sun, 12 Jan 2025 06:46:12 +0000
Subject: [PATCH] Enable capture capabilities

This patch adds permission of capture audio output,
which is required by echo-reference device to capture
output for looopback purpose.

Test Done:
-loopback verification with kitchen sink app

Tracked-On: OAM-116907
Signed-off-by: sadhansx <sadhanx.siddha.mohanty@intel.com>
Signed-off-by: sgnanase <sundar.gnanasekaran@intel.com>
---
 data/etc/privapp-permissions-platform.xml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/etc/privapp-permissions-platform.xml b/data/etc/privapp-permissions-platform.xml
index 880f30c6cdc0..6580437b1e1c 100644
--- a/data/etc/privapp-permissions-platform.xml
+++ b/data/etc/privapp-permissions-platform.xml
@@ -25,6 +25,10 @@ applications that come with the platform
         <permission name="android.permission.READ_PRIVILEGED_PHONE_STATE"/>
     </privapp-permissions>
 
+    <privapp-permissions package="com.google.android.car.kitchensink">
+        <permission name="android.permission.CAPTURE_AUDIO_OUTPUT"/>
+    </privapp-permissions>
+
     <privapp-permissions package="com.android.angle">
         <permission name="android.permission.WRITE_SECURE_SETTINGS"/>
     </privapp-permissions>
-- 
2.34.1

